<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STSettingClass</class>
 <widget class="QDialog" name="STSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STSetting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widTitle" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #434555;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>系统设置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>448</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_17" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbClose">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{border-image: url(:/SoftTerminal/images/close.png);}

QPushButton:hover:!pressed {
	
	background-color: #ff6741;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{border-bottom: 1px solid #e3e3e3;border-left: 1px solid #e3e3e3;
border-right: 1px solid #e3e3e3;}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget{background-color: #ffffff;}
QTabWidget::pane{border:0px;}

/*QTabWidget::tab-bar{alignment:center;}*/

#tabWidget QTabBar::tab
{
    min-width: 75px;
    min-height: 28px;
	font: 10pt &quot;微软雅黑&quot;;
}


#tabWidget QTabBar::tab:selected
{
    border-bottom: 2px solid #00c36e;
	color:#00c36e;
}

#tabWidget QTabBar::tab:!selected
{
    border-bottom: 0px;
	color: rgb(67, 69, 85);
}
</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="twServer">
         <attribute name="title">
          <string>服务器</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>94</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QWidget#widget_8{border: 0px;border-bottom: 1px solid #e3e3e3;}
</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(67, 69, 85);</string>
                  </property>
                  <property name="text">
                   <string>　服务器地址　</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_9" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>10</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>10</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Adobe Arabic</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 0px;border-left: 1px solid #e3e3e3;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leServerIP">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(67, 69, 85);border: 0px;border-bottom: 1px solid #e3e3e3;
</string>
               </property>
               <property name="maxLength">
                <number>20</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>94</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pbConfirm">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:pressed {
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:!pressed {
	
	background-color: rgb(109, 206, 160);
}</string>
               </property>
               <property name="text">
                <string>确　定</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>94</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="twDevice">
         <attribute name="title">
          <string>设备</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>34</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_10" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QWidget#widget_10{border: 0px;border-bottom: 1px solid #e3e3e3;}
</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(67, 69, 85);</string>
                  </property>
                  <property name="text">
                   <string>摄像头</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_11" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>10</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>10</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Adobe Arabic</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 0px;border-left: 1px solid #e3e3e3;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbxCamera">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::WheelFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox
{
	color: rgb(67, 69, 85);
	border: 0px;
	border-bottom: 1px solid #e3e3e3;
}
QComboBox::drop-down
{
	background-color: white;
	min-width: 30px;
}
QComboBox::down-arrow {
	image: url(:/SoftTerminal/images/triangle_focus.png);
}
QComboBox QAbstractItemView
{
border: 1px solid #e3e3e3;
}
QComboBox QAbstractItemView::item
{
height: 30px;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widPreview" native="true">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>180</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/SoftTerminal/images/camoff.png);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3"/>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_12" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QWidget#widget_12{border: 0px;border-bottom: 1px solid #e3e3e3;}
</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(67, 69, 85);</string>
                  </property>
                  <property name="text">
                   <string>麦克风</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_13" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>10</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>10</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Adobe Arabic</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 0px;border-left: 1px solid #e3e3e3;
</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbxMic">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::WheelFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox
{
	color: rgb(67, 69, 85);
	border: 0px;
	border-bottom: 1px solid #e3e3e3;
}
QComboBox::drop-down
{
	background-color: white;
	min-width: 30px;
}
QComboBox::down-arrow {
	image: url(:/SoftTerminal/images/triangle_focus.png);
}
QComboBox QAbstractItemView
{
border: 1px solid #e3e3e3;
}
QComboBox QAbstractItemView::item
{
height: 30px;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>35</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pbConfirmDevice">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:pressed {
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:!pressed {
	
	background-color: rgb(109, 206, 160);
}</string>
               </property>
               <property name="text">
                <string>确　定</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>34</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/softterminal.qrc"/>
 </resources>
 <connections/>
</ui>
